pallets/corporate-actions/src/ballot.rs

```diff
@@ -262,7 +262,7 @@ decl_storage! {
         ///
         /// User must enter 0 vote weight if they don't want to vote for a choice.
         pub Votes get(fn votes):
-            double_map hasher(blake2_128_concat) CAId, hasher(blake2_128_concat) IdentityId =>
+            double_map hasher(blake2_128_concat) CAId, hasher(identity) IdentityId =>
```

Comment:
Can we use Twox for the `CAId`s? One component is an incrementing index, and the `Ticker` needs to actually exist and be connected to a CA (`CorporateActions` map ensures this and uses `blake2_128_concat`), so untrusted users cannot select it "en masse".

Reply:
True that users can not select it "en masse" without paying the protocol fee but governance might remove the protocol in future and that'll make it cheaper to attack. Given that the performance difference between hashers is so small, I'd rather play it safe.

Reply:
True that users can not select it "en masse" without paying the protocol fee but governance might remove the protocol in future and that'll make it cheaper to attack. Given that the performance difference between the different hashers is so small, I'd rather play it safe.